vxfw/vxfw.go

```diff
@@ -369,6 +375,19 @@ func (f *focusHandler) focusWidget(app *App, w Widget) error {
 	return nil
 }
 
+// tryHandleEvent calls HandleEvent on w, if w is an EventHandler
+// It returns true if w is an EventHandler, along with the command and error resulting from
+// HandleEvent, if any.
+func tryHandleEvent(w Widget, event vaxis.Event, phase EventPhase) (bool, Command, error) {
```

Comment:
It doesn't look like the `bool` returned is used anywhere, do you have plans for it or could we remove it from the return signature?

Reply:
I'm going to take out the boolean return value and force push. brb.